src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -360,7 +361,11 @@ private void sendErrorMessage(
         ErrorCode errorCode
     ) {
         try {
-            sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
+            String message =
+                cause instanceof OcrValidationException
+                    ? ((OcrValidationException) cause).getDetailMessage()
+                    : cause.getMessage();
+            sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, message);
```

Comment:
As of the comment below for me looks better:
String message = cause.getMessage();
if (cause instanceof OcrValidationException) {
    message += cause.getDetailMessage();
}

Reply:
changed the logic

Reply:
not sure I understand what you mean

Reply:
refresh the page, I changed with if.